cluster/gce/gci/shutdown.sh

```diff
@@ -16,7 +16,7 @@
 
 # A script that let's gci preemptible nodes gracefully terminate in the event of a VM shutdown.
 preemptible=$(curl "http://metadata.google.internal/computeMetadata/v1/instance/scheduling/preemptible" -H "Metadata-Flavor: Google")
-if [ ${preemptible} == "TRUE" ]; then
+if [ "${preemptible}" == "TRUE" ]; then
```

Comment:
```
In ./cluster/gce/gci/shutdown.sh line 19:
if [ ${preemptible} == "TRUE" ]; then
     ^-- SC2086: Double quote to prevent globbing and word splitting.
```